src/components/encoding-pane/filter-shelf.scss

```diff
@@ -22,9 +22,21 @@ $radius: 3px;
   }
 }
 
-.header {
```

Comment:
I wonder why we have to remove header here.  (Why is it relevant to adding a timeUnit filter?)